build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "spark-avro"
 
-organization := "com.databricks"
+organization := "com.kainos"
 
 scalaVersion := "2.10.5"
```

Comment:
Good spot. I don't believe this matters as they are both 2.10. I'll change this one to keep it consistent